### PR TITLE
refactor(console): new get started page

### PR DESCRIPTION
**Pull request by charIeszhao**
## Summary

Refactor the get-started page in Admin Console, including new application integration guides and layout adjustments.

## Testing

Locally tested

image

## Checklist

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/console/src/containers/ErrorBoundary/index.tsx [hunk 1]
```diff
@@ -38,6 +38,7 @@ class ErrorBoundary extends Component<Props, State> {
   render() {
     const { children, t } = this.props;
     const { error } = this.state;
+    console.log(error);
```
Comment (simeng-li) [inline packages/console/src/containers/ErrorBoundary/index.tsx:41]: not expected?

File: packages/console/src/ds-components/Button/index.tsx [hunk 2]
```diff
@@ -122,25 +125,42 @@ export function LinkButton({
...
-    <a
-      className={classNames(styles.button, styles[type], styles[size], className)}
-      {...rest}
-      {...(Boolean(targetBlank) && {
+  const { getTo } = useTenantPathname();
+  const props = useMemo(
+    () => ({
+      ...rest,
+      className: classNames(styles.button, styles[type], styles[size], className),
+      ...(Boolean(targetBlank) && {
```
Comment (simeng-li) [inline packages/console/src/ds-components/Button/index.tsx:137]: Can we simply pop up the target and ref props to the outer scope?

File: packages/console/src/ds-components/Button/index.tsx [hunk 3]
```diff
@@ -122,25 +125,42 @@ export function LinkButton({
...
+      ),
+    [title]
+  );
+
+  return !href || isAbsoluteUrl(href) ? (
+    <a {...props} href={href}>
+      {innerElement}
     </a>
+  ) : (
+    <Link {...props} to={getTo(href)}>
```
Comment (simeng-li) [inline packages/console/src/ds-components/Button/index.tsx:162]: Looks like not all the LinkProps are supported by the LinkButton wrapper. e.g. `replace`, and `state`.

File: packages/console/src/containers/ErrorBoundary/index.tsx (hunk 1 above)
Comment (charIeszhao) [reply packages/console/src/containers/ErrorBoundary/index.tsx:41]: No, I'll remove it. Thanks

File: packages/console/src/ds-components/Button/index.tsx (hunk 3 above)
Comment (charIeszhao) [reply packages/console/src/ds-components/Button/index.tsx:162]: The Button component will be refactored in another PR. Linear task created
https://linear.app/silverhand/issue/LOG-7011/refactor-button-component-in-admin-console

File: packages/console/src/ds-components/Button/index.tsx (hunk 2 above)
Comment (charIeszhao) [reply packages/console/src/ds-components/Button/index.tsx:137]: Will refactor Button component in a separated task.
https://linear.app/silverhand/issue/LOG-7011/refactor-button-component-in-admin-console

File: packages/console/src/pages/GetStarted/index.tsx [hunk 4]
```diff
@@ -1,110 +1,147 @@
...
-    try {
-      await update({ getStartedHidden: true });
-      // Navigate to next menu item
-      navigate('/dashboard');
-    } finally {
-      setIsUpdating(false);
-    }
-  };
+  const featuredAppGuides = useMemo(
+    () => getStructuredMetadata().featured.slice(0, 4),
```
Comment (gao-sun) [inline packages/console/src/pages/GetStarted/index.tsx:44]: better to leave a comment here for why we slice for 4 items

File: packages/console/src/pages/GetStarted/index.tsx [hunk 5]
```diff
@@ -1,110 +1,147 @@
...
-    setIsUpdating(true);
+  const { tenantEndpoint } = useContext(AppDataContext);
+  const [selectedGuide, setSelectedGuide] = useState<SelectedGuide>();
+  const [_, getStructuredMetadata] = useAppGuideMetadata();
+  const [showCreateForm, setShowCreateForm] = useState<boolean>(false);
+  const theme = useTheme();
+  const isLightMode = theme === Theme.Light;
+  const PreviewIcon = isLightMode ? CheckPreview : CheckPreviewDark;
+  const SocialIcon = isLightMode ? Social : SocialDark;
+  const RbacIcon = isLightMode ? CreateRole : CreateRoleDark;
```
[the remark this answers is not in the document]
Comment (gao-sun) [reply packages/console/src/pages/GetStarted/index.tsx:41]:
```tsx
const icons = {
  [Theme.Light]: { PreviewIcon, SocialIcon },
  [Theme.Dark]: { PreviewIcon, SocialIcon },
};

function GetStarted() {
  const { PreviewIcon, SocialIcon } = icons[theme];
}
```

[the remark this answers is not in the document]
Comment (charIeszhao) [reply packages/console/src/pages/GetStarted/index.tsx]: updated

File: packages/console/src/pages/GetStarted/index.tsx (hunk 4 above)
Comment (charIeszhao) [reply packages/console/src/pages/GetStarted/index.tsx:47]: comment added

**Review by gao-sun — APPROVED** [review]

File: packages/console/src/pages/GetStarted/index.tsx [hunk 6]
```diff
@@ -1,110 +1,150 @@
...
-
-    try {
-      await update({ getStartedHidden: true });
-      // Navigate to next menu item
-      navigate('/dashboard');
-    } finally {
-      setIsUpdating(false);
-    }
-  };
+  // Only need 4 featured guides at most, since we only show 1 row of the guide cards by design
```
Comment (gao-sun) [inline packages/console/src/pages/GetStarted/index.tsx:45]: 1 or 2 rows?

**Pull request merged**